cmd/create_data_stream.go

```diff
@@ -80,14 +83,21 @@ func createDataStreamCommandAction(cmd *cobra.Command, args []string) error {
 			},
 			Validate: survey.Required,
 		},
-		{
+	}
+
+	sv, err := semver.NewVersion(manifest.SpecVersion)
+	if err != nil {
+		return fmt.Errorf("failed to obtain spec version from package manifest in \"%s\"", packageRoot)
```

Comment:
Wrap the error to have more context if this fails.
```suggestion
		return fmt.Errorf("failed to obtain spec version from package manifest in \"%s\": %w", packageRoot, err)
```